types/carbon-components-react/lib/components/OverflowMenu/OverflowMenu.d.ts

```diff
@@ -11,7 +11,7 @@ import {
 type GetMenuOffsetFn = ((menuBody: HTMLElement, direction: Direction, trigger?: HTMLElement, flip?: boolean) => (MenuOffsetData | undefined));
 export declare const getMenuOffset: GetMenuOffsetFn;
 
-type ExcludedAttributes = "aria-expanded" | "aria-haspopup" | "aria-label" | "onBlur" | "onClick" | "onKeyDown" | "onKeyPress" | "role";
+type ExcludedAttributes = "aria-expanded" | "aria-haspopup" | "aria-label" | "onBlur" | "onKeyPress" | "role";
```

Comment:
LGTM after the actual `onClick` property is removed from `InheritedProps`. I think they merged both events in one handler in older versions.

Reply:
Good deal. Fixed.